types/rlp_unmarshal.go

```diff
@@ -60,12 +80,24 @@ func (b *Block) UnmarshalRLPFrom(p *fastrlp.Parser, v *fastrlp.Value) error {
 		return err
 	}
 
-	for _, txn := range txns {
```

Comment:
Why not `range`?

Reply:
Because it has a little bit complicated logic, look at line 90

Reply:
We are incrementing the index manually to skip the type right?